WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,5 +1,4 @@
-* [*] Block editor: Replacing the media for an image set as featured prompts to update the featured image [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3930]
-* [*] Block editor: Fixed an issue in sharing images from other apps [https://github.com/wordpress-mobile/WordPress-Android/pull/15864]
-* [**] Stats revamp: Adding a new settings button to the top app bar to manage Stats Insights Cards [https://github.com/wordpress-mobile/WordPress-Android/pull/15891]
-* [***] Block editor: Font size and line-height support for text-based blocks used in block-based themes [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4519]
-
+- See text blocks in the block editor adjust with changing font size and line height.
```

Comment:
@geriux I'm guessing this will need a similar update like the iOS one, as you mentioned internally?

Reply:
👋 @mokagio !

FYI: The [relevant above PR](https://github.com/wordpress-mobile/WordPress-Android/pull/15933) is now merged.